File: HCS.Client/ClientApp/app/models/consumption-norm.ts
export class ConsumptionNorm {
    id: number;
    amount: number;
    consumedUtilityId: number;
    utilityName: string;

    constructor(consumedUtilityId: number, utilityName: string) {
        this.consumedUtilityId = consumedUtilityId;
        this.utilityName = utilityName;
    }
}

File: HCS.Core/Domain/ConsumptionNorm.cs
namespace HCS.Core.Domain
{
    public class ConsumptionNorm
    {
        public int Id { get; set; }
        public double Amount { get; set; }
        public int ConsumedUtilityId { get; set; }
        public ConsumedUtility ConsumedUtility { get; set; }
        public int OccupantId { get; set; }
    }
}
